components/ui/Typography.jsx

```diff
@@ -1,35 +1,30 @@
+
 export const Heading1 = ({ children, className }) => {
     return (
-        <h1 className={`md:text-23 md:leading-15.33 ${className}`}>{children}</h1>
+        <h1 className={`font-medium md:font-normal md:text-14.75 text-7.25 md:leading-22 leading-11 -tracking-0.02 ${className}`}>{children}</h1>
     )
 }
 
 export const Heading2 = ({ children, className }) => {
     return (
-        <h2 className={`font-bold md:text-18.25 md:leading-22.75 leading-18.5 text-9.5 ${className}`}>{children}</h2>
+        <h2 className={`font-normal md:text-11.75 md:leading-14.75 leading-7.15 text-6.5 text-white -tracking-0.015 md:-tracking-0.02 ${className}`}>{children}</h2>
     )
 }
```

Comment:
Instead of having different heading components which are only differentiated with numbers, we can have one and just switch variants according to their type. Here classname package would also play a huge role.
```javascript 
const Heading = ({ children, className, type = 'one' }) {
    /* we can use here classname package to handle the variant for each heading type */
    const headingClassname = classname('default classes', { 'conditional class': type === 'one' });
}
```